.github/workflows/run-check.yml

```diff
@@ -0,0 +1,29 @@
+name: Fetch metrics
+
+on: [pull_request]
+
+jobs:
+  get_metrics:
+    runs-on: ubuntu-latest
+    name: Get repository metrics
+    steps:
+      - uses: paritytech/metrics@pull/stadistics
+        name: Fetch polkadot-fellows/runtimes stadistics
```

Comment:
```suggestion
        name: Fetch polkadot-fellows/runtimes statistics
```

:D

Reply:
Ah it's the same PR as before, nevermind.

Reply:
I'm gonna delete that file at the end, it just for testing.

I changed the name of the PR tho, so it should have fixed that